packages/genui/playground/src/pages/DemosPage.css

```
@@ -510,6 +510,29 @@
+  font-size: 13px;
+  line-height: 1.65;
+}
+
+.openuiCodeEditor .cm-content {
+  padding: 18px 22px 26px;
+}
+
+.openuiCodeEditor .cm-line {
+  padding: 0 0 12px;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)